Description: Remove 'dns-internal' internal library
 Our haskell-devscripts cannot handle Haskell packages with more than one
 libraries. Remove the 'dns-internal' internal library and move its sources
 under other-modules.
Author: Ilias Tsitsimpis <iliastsi@debian.org>

--- a/dns.cabal
+++ b/dns.cabal
@@ -20,41 +20,6 @@ Tested-With:            GHC == 8.0.2
                       , GHC == 8.6.5
                       , GHC == 8.8.1
 
-Library dns-internal
-  Default-Language:     Haskell2010
-  GHC-Options:          -Wall
-  Hs-Source-Dirs:       internal
-  Exposed-Modules:      Network.DNS.Imports
-                        Network.DNS.Types.Internal
-                        Network.DNS.Types.Resolver
-                        Network.DNS.Resolver.Internal
-                        Network.DNS.Decode.Parsers
-                        Network.DNS.Decode.Internal
-                        Network.DNS.Encode.Builders
-                        Network.DNS.Encode.Internal
-                        Network.DNS.StateBinary
-                        Network.DNS.Memo
-                        Network.DNS.Base32Hex
-  Build-Depends:        base
-                      , array
-                      , async
-                      , attoparsec
-                      , auto-update
-                      , base16-bytestring
-                      , base64-bytestring
-                      , bytestring
-                      , case-insensitive
-                      , containers
-                      , cryptonite
-                      , hourglass
-                      , iproute
-                      , mtl
-                      , network
-                      , psqueues
-  if os(windows)
-    C-Sources:        cbits/dns.c
-    Extra-Libraries:  iphlpapi
-
 Library
   Default-Language:     Haskell2010
   GHC-Options:          -Wall
@@ -68,8 +33,19 @@ Library
                         Network.DNS.Encode
                         Network.DNS.IO
   Other-Modules:        Network.DNS.Transport
-  Build-Depends:        dns-internal
-                      , base >= 4 && < 5
+                        Network.DNS.Imports
+                        Network.DNS.Types.Internal
+                        Network.DNS.Types.Resolver
+                        Network.DNS.Resolver.Internal
+                        Network.DNS.Decode.Parsers
+                        Network.DNS.Decode.Internal
+                        Network.DNS.Encode.Builders
+                        Network.DNS.Encode.Internal
+                        Network.DNS.StateBinary
+                        Network.DNS.Memo
+                        Network.DNS.Base32Hex
+  Hs-Source-Dirs:       . internal
+  Build-Depends:        base >= 4 && < 5
                       , array
                       , async
                       , attoparsec
@@ -77,6 +53,7 @@ Library
                       , base16-bytestring
                       , base64-bytestring
                       , bytestring
+                      , case-insensitive
                       , containers
                       , cryptonite
                       , hourglass
@@ -94,7 +71,6 @@ Test-Suite network-tests
   Other-Modules:        LookupSpec
                         IOSpec
   Build-Depends:        dns
-                      , dns-internal
                       , base
                       , hspec
                       , network
@@ -110,7 +86,6 @@ Test-Suite spec-tests
                         DecodeSpec
                         RoundTripSpec
   Build-Depends:        dns
-                      , dns-internal
                       , QuickCheck >= 2.9
                       , base
                       , bytestring
